challenge.rb

```diff
@@ -21,11 +26,15 @@ def is_of_age? age, minimum_age
 # This question will be graded independently of question one.
 # Remember: `p`, `print`, and `puts` don't return their arguments.
 
-def okay_to_drink? age, drinking_age
-  # Replace this comment with your code.
+def okay_to_drink?(age, drinking_age)
+  if (is_of_age?(age, drinking_age) == true)
+    puts "Come on in."
+  else
+    puts "I'm sorry, #{age} isn't old enough to dwwink beews. The min age is #{drinking_age}."
+  end
```

Comment:
Supposed to return instead of print, but otherwise correct. But per my comment in Q1, this could also have been written a bit more tersely. The statement `x == true` is redundant. If `x` is true, the expression evaluates to `true`, and if `x` is false, the expression evaluates to `false`. In short, there's no reason why we couldn't write `x` instead of `x == true`. Or, in your case, just writing `if is_of_age?(age, drinking_age)`